lib/src/tileset/tile.dart

```diff
@@ -47,7 +47,10 @@ class Tile {
   static Tile parse(Parser parser) {
     return Tile(
       localId: parser.getInt('id'),
-      type: parser.getStringOrNull('type'),
+
+      /// Tiled 1.9 "type" has been moved to "class"
+      type: parser.getStringOrNull('type') ?? parser.getStringOrNull('class'),
```

Comment:
Just to make it consistent with #48, you can make it check for `class` first, and then `type`. As more and more people will migrate to Tiled 1.9, chances of finding `class` property instead of `type` are more likely.

Reply:
Fixed the ordering.

I think with Tiled 1.9 you can add `class` to nearly anything, which is why they did the rename. We'd probably need to got through and test this for a bunch of other objects, but Layers for one could be useful. Although it raises the question, should we add a `tiledClass` attribute or keep calling them `type` even though they property in the new Tiled UI is "Class", I guess they named it that because layers already have a "type" which means something else.

> In addition, the “Type” property previously available only for objects and tiles is now available for all data types as the new “Class” property. For consistency, this value is written out as “class” also for objects and tiles, but a project-wide compatibility option is provided to make it still write out as “type”

Reply:
For `TiledObject` class I kept `type` attribute as it was and just added a getter called `class_` which returns `type`. That way it won't break anyone's existing code and if someone using tiled with newer map files, tries to find class attribute, `class_` will pop-up in auto-complete list.